public/client.js

```diff
@@ -1,9 +1,10 @@
-// $(function () {
-// 	$.get('/trucks', function (trucks) {
-// 		var truckList = [];
-// 		trucks.forEach(function (truck) {
-// 			truckList.push('<li><a href="/trucks/' + truck.name + '/">' + truck.name + '</a></li>')
-// 		});
-// 		$('.truck-list').append(truckList);
-// 	});
-// });
\ No newline at end of file
+ $(function () {
+ 	$.get('/trucks', function (trucks) {
+ 		var truckList = [];
+		var truckObject = JSON.parse(trucks);
```

Comment:
This step actually causes the client code to not work. Instead, just loop through trucks and it will work fine.